central/reports/service/v2/service_impl_test.go

```diff
@@ -1455,12 +1433,7 @@ func (s *ReportServiceTestSuite) TestPostViewBasedReport() {
 				Type: apiV2.ReportRequestViewBased_VULNERABILITY,
 				Filter: &apiV2.ReportRequestViewBased_ViewBasedVulnReportFilters{
 					ViewBasedVulnReportFilters: &apiV2.ViewBasedVulnerabilityReportFilters{
-						ImageTypes: []apiV2.ViewBasedVulnerabilityReportFilters_ImageType{
-							apiV2.ViewBasedVulnerabilityReportFilters_DEPLOYED,
```

Comment:
nit: You can remove this test case now that we are not giving an option to select between DEPLOYED and WATCHED images.